src/common/components/ui/OpenShiftVersionDropown.tsx

```diff
@@ -82,17 +85,30 @@ export const OpenShiftVersionDropdown = ({
 
   return (
     <FormGroup fieldId={fieldId} label={t('ai:OpenShift version')} isRequired>
-      <Dropdown
-        {...field}
-        name={name}
-        id={fieldId}
-        onSelect={onSelect}
-        dropdownItems={dropdownItems}
-        toggle={toggle}
-        isOpen={isOpen}
-        className="pf-u-w-100"
-      />
-      <HelperText style={{ display: 'inherit' }}>{helperText}</HelperText>
+      <Flex direction={{ default: 'column', lg: 'row' }}>
+        <FlexItem flex={{ default: 'flex_1' }}>
+          <Dropdown
+            {...field}
+            name={name}
+            id={fieldId}
+            onSelect={onSelect}
+            dropdownItems={dropdownItems}
+            toggle={toggle}
+            isOpen={isOpen}
+            className="pf-u-w-100"
+          />
+          <HelperText style={{ display: 'inherit' }}>{helperText}</HelperText>
+        </FlexItem>
+        <FlexItem align={{ default: 'alignRight' }}>
+          <ExternalLink
+            href={
```

Comment:
- Please define a constant as we do for other links etc so it's easier to find and update.
- I don't think this is the correct link, this one looks to be internal page for a specific release.

Reply:
- Ok, I've added a constant
- About the link is not clear which is the correct link. I've asked in https://issues.redhat.com/browse/MGMT-11900